### DISCUSSION TITLE
Minimizing the keyboard after exiting the app

### DISCUSSION THREAD

**Discussion by ghost** [AppGround-io/bluetooth-keyboard-and-mouse-support#281] [Bugs & Problems]
I suggest minimizing the keyboard after exiting the app, as the keyboard remains visible over other apps, interfering with its use.

**Comment by lubu8** [comment]
Yes please, as it makes it difficult to use gestures to control the phone. The keyboard just stays there upon leaving the app. To make it go away, I have to either remember to close it before I exit the app, or go back into the app, which strangely closes the keyboard, only to exit again. Is there an easier way? I couldn't find the appropriate setting.
